tools/write_inputs.m

```diff
@@ -20,10 +20,18 @@
 % This script is run by the bash script da_inp.sh.
 % It used to generate the necessary input files for uDALES.
 
-expnr = '901';
 %
+if exist('expnr', 'var')
+    expnr = sprintf('%03d',expnr);
+else
+    disp('Error: No input argument provided. The script will terminate.');
+    return;  % Terminate the script
```

Comment:
Consider `error('Error: No input argument provided. The script will terminate.');`

or even better, throw an exception:

```suggestion
    ME = MException('InputError', 'No input argument provided');
    throw(ME)
```

Reply:
Used error('Error: No input argument provided. The script will terminate.');